### PR TITLE
worked on ticket request information

### PR DESCRIPTION
**Pull request by kkadaru**
# Description

This PR includes the following proposed change(s):

- added ticket request information to staff portal

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/src/app/components/ticket-request/ticket-request.component.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            <section>
+                <div class="row">
+                    <div class="col-md-6">
+                        <mat-form-field class="w-100">
+                            <mat-label>Request type</mat-label>
+                            <input matInput />
+                            <!-- <mat-error>{{ "error.required" | translate }}</mat-error> -->
+                        </mat-form-field>
+                    </div>
+                    <div class="col-md-4">
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/components/ticket-request/ticket-request.component.html:48]: remove the commented out validation

**Review by lfdgov — APPROVED** [review]

**Review by KDNeufeld — CHANGES_REQUESTED** [review]
This branch is not sourced from develop but rather main.  Please rebase your changes onto develop into order to merge into develop.

**Comment by Sharkww** [comment]
suggest to add the JIRA ticket TCVP-1207 to the PR title

**Review by KDNeufeld — APPROVED** [review]
The branch as successfully rebased onto develop.  Great work, thanks.

**Pull request merged**